scrape/scrape_test.go

```diff
@@ -300,6 +300,7 @@ func TestScrapePoolReload(t *testing.T) {
 func TestScrapePoolAppender(t *testing.T) {
 	cfg := &config.ScrapeConfig{}
 	app := &nopAppendable{}
+	ctx := context.Background()
```

Comment:
We can just use `context.Background()` in those 2 places, I don't think a new variable is required.